index.js

```diff
@@ -5,6 +5,9 @@ const zlib = require('zlib')
 const pump = require('pump')
 const sts = require('string-to-stream')
 const mimedb = require('mime-db')
+const isStream = require('is-stream')
+const intoStream = require('into-stream')
+const maybeUnzip = require('unzip-if-zip')
```

Comment:
I'm not very comfortable in adding a dependency on this module. Are there any alternatives?

Reply:
yeah it seemed sus. I checked out the source, it's fairly simple to re-implement. probably worth doing just to add support for brotoli too.